flink-core/src/main/java/org/apache/flink/streaming/api/transformations/StreamTransformation.java

```diff
@@ -100,6 +96,9 @@
 @Internal
 public abstract class StreamTransformation<T> {
```

Comment:
@aljoscha what is your opinion about this change? Should we simply move the class into the same package or should we deprecate the old location and have a new location for streaming related classes in core?

Reply:
I think it's okay to keep it in the current package. An alternative would be `org.apache.flink.api.dag` or something like it.